src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1060,6 +1082,16 @@ void deleteGraphPartition(std::string graphID, std::string partitionID) {
     instance_logger.log("Graph partition and centralstore files are now deleted", "info");
 }
 
+void removeGraphFragments(std::string graphID) {
+    /**
```

Comment:
Please move this multi-line comment above the function signature. Please change the multi-line comment following these [guidelines](https://developer.lsst.io/v/DM-5063/docs/cpp_docs.html#function-method-definitions).

Reply:
Followed the guidelines and moved the multi-line comment above the function signature. Used @param to before describing each input parameter. Removed "INPUT" and "OUTPUT" parts of the comment